.travis/tests.sh

```diff
@@ -1,27 +1,30 @@
 #!/bin/bash
 
+set -ve
+
 if [[ ${TEST} = 'main' ]]
 then
 
+    cp .travis/environments/travis/private.yml .travis/environments/travis/vault.yml
+
     test_syntax() {
-        ansible-playbook -i .travis/environments/travis/inventory.ini ansible/deploy_stack.yml --syntax-check
+        COMMCARE_CLOUD_ENVIRONMENTS=.travis/environments commcare-cloud travis deploy-stack --branch=FETCH_HEAD  --skip-check --quiet --syntax-check
     }
 
     test_localsettings() {
-        cp .travis/environments/travis/private.yml .travis/environments/travis/vault.yml
         COMMCARE_CLOUD_ENVIRONMENTS=.travis/environments commcare-cloud travis deploy-stack --branch=FETCH_HEAD  --skip-check --quiet --tags=commcarehq
         sudo python -m py_compile /home/cchq/www/travis/current/localsettings.py
     }
 
     test_help_cache() {
-        diff <(ansible -h) commcare-cloud/commcare_cloud/help_cache/ansible.txt
-        diff <(ansible-playbook -h) commcare-cloud/commcare_cloud/help_cache/ansible-playbook.txt
+        diff <(ansible -h | sed "s#$HOME#\~#g") src/commcare_cloud/help_cache/ansible.txt
+        diff <(ansible-playbook -h | sed "s#$HOME#\~#g") src/commcare_cloud/help_cache/ansible-playbook.txt
     }
```

Comment:
Hey @snopoke so actually I think this test can be removed entirely. `make` now auto-generates these files (https://github.com/dimagi/commcare-cloud/blob/master/Makefile#L3-L7) so `./tests/test_command_docs.sh` (which runs `make` and then sees if there's a diff) is already testing this now.

I guess I would have noticed and removed this test back when I added that, if it had been failing the build!